src/main/java/TulipTask.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class TulipTask {
```

Comment:
Consider using another name such as "Tulip" because "TulipTask" makes give the impression that TuilpTask inherits from Task class